tests/bsim/bluetooth/ll/bis/Kconfig

```diff
@@ -1,11 +1,18 @@
-# Copyright 2024 Nordic Semiconductor ASA
+# Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
 config TEST_LL_INTERFACE
 	bool "Test Broadcast ISO using internal LL interface"
 	help
 	  Test Broadcast ISO using internal LL interface.
 
+config BT_ISO_TEST_INTERLEAVED
+	bool "Test ISO interleaved packing"
```

Comment:
Ditto; please do not use `BT_ISO` for non-stack Kconfig options

Reply:
Updated